zebra/zebra_mroute.c

```diff
@@ -65,7 +65,6 @@ void zebra_ipmr_route_stats(ZAPI_HANDLER_ARGS)
 	stream_put_in_addr(s, &mroute.sg.src);
 	stream_put_in_addr(s, &mroute.sg.grp);
 	stream_put(s, &mroute.lastused, sizeof(mroute.lastused));
-	stream_putl(s, suc);
```

Comment:
part of the pim rfc states that we need to know about data flow.  This is the mechanism between zebra ( the one asking the kernel for the data ) and pim that needs it to make control plane decisions.  While failure is rare and I haven't coded it in yet.  I at least had a plan to eventually get there( it would be awful nice if pim were able to know that the query was attempted yet it failed and able to display that information to end users ).   I don't personally ahve a problem with fixing signed/unsigned issues.  but perhaps we could keep this?

Reply:
sure put it back